frictionless/core.py

```diff
@@ -189,3 +192,15 @@ def resource_descriptor(dfile):
     return dict(**package_meta,
                 **{"resources": resources,
                    "created": datetime.now(timezone.utc).isoformat()})
+
+def build_datapackage_from_id(source: str) -> Dict:
+    """Build a datapackaage from a PUDL source."""
+    dp = {}
+    dp["profile"] = "data-package"
+    dp["homepage"] = "https://catalyst.coop/pudl/",
+    dp["keywords"] = KEYWORDS_BY_SOURCE[source]
+    dp["sources"] = SOURCES[source]
+    
+    cids = CONTRIBUTORS_BY_SOURCE[source]
```

Comment:
`pudl-zenodo-storage/frictionless/contributors.py` only contains metadata for Catalyst Cooperative. I figure we should include contributors by source. What do you think @zaneselvans?

Reply:
Yeah I agree. Different individuals have contributed to different data sources and we're tracking that information loosely already in the PUDL metdata, so why not use it here as well.